cmd/ptcp/ptcp.go

```diff
@@ -0,0 +1,171 @@
+package ptcp
+
+/* ptcp is a cp-like tool that can copy files in and out of the Pairtree structure.
+Unlike Linux's cp, the default is recursive */
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	error_msgs "github.com/UCLALibrary/pt-tools/pkg/error-msgs"
+	"github.com/UCLALibrary/pt-tools/pkg/pairtree"
+	"github.com/UCLALibrary/pt-tools/utils"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+)
+
+var (
+	overwrite bool
+	tar       bool
+	subpath   string
+	ptRoot    string
+	logFile   string      = "logs.log"
+	Logger    *zap.Logger = utils.Logger(logFile)
+	src       string      = ""
+	dest      string      = ""
+)
+
+func initFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVarP(&ptRoot, "pairtree", "p", "", "Set pairtree root directory")
+	cmd.Flags().BoolVarP(&overwrite, "d", "d", false, "Overwrite target files")
+	cmd.Flags().StringVarP(&subpath, "n", "n", "", "Create subpath to or rename the file or path")
+	cmd.Flags().BoolVarP(&tar, "a", "a", false, "Produce a tar/gzipped output or unpack a tar/gzipped")
+}
+
+func Run(args []string, writer io.Writer) error {
+	var err error
+
+	var rootCmd = &cobra.Command{
+		Use:   "ptrm [PT_ROOT] [ID] [subpath/to/file.txt]",
+		Short: "ptrm is a tool to remove Pairtree objects, files, and directores",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// If the root has not been set yet check the ENV vars
+			if ptRoot == "" {
+
+				if envVar := os.Getenv("PAIRTREE_ROOT"); envVar != "" {
+					ptRoot = envVar
+				} else {
+					fmt.Fprintln(writer, error_msgs.Err7)
+					return error_msgs.Err7
+				}
+			}
+
+			numArgs := len(args)
+			if numArgs < 2 {
+				fmt.Fprintln(writer, "Please provide a source and destination for copied files")
+				Logger.Error("There are not enough arguments to ptcp",
+					zap.Error(error_msgs.Err9))
+
+				return error_msgs.Err9
+			}
+
+			if numArgs == 2 {
+				// Extract the ID and the subpath from the arguments
+				src = args[numArgs-2]
+				dest = args[numArgs-1]
+			} else {
+				fmt.Fprintln(writer, "Too many arguments were provided to ptcp")
+				Logger.Error("Error parsing ptcp",
+					zap.Error(error_msgs.Err8))
```

Comment:
Seems like this could go on line above? Are there other places that could use 120 chars on one line instead of wrapping?